pipelines/go/bump.yaml

```diff
@@ -50,5 +49,16 @@ pipeline:
         go mod graph
       fi
 
+      # Packages and replaces are optional, so if not given, we don't pass them.
+      packagesString=""
+      if [ ! -z "${{inputs.deps}}" ]; then
+        packagesString="--packages ${{inputs.deps}}"
+      fi
+
+      replacesString=""
+      if [ ! -z "${{inputs.replaces}}" ]; then
+        replacesString="--replaces ${{inputs.replaces}}"
+      fi
+
       # We use the --tidy flag to run go mod tidy before and after in some cases (if old versions of go are used, we need to update the go.mod format)
-      gobump --packages "${{inputs.deps}}" --replaces "${{inputs.replaces}}" --tidy=${{inputs.tidy}} --skip-initial-tidy=${{inputs.skip-initial-tidy}} --show-diff=${{inputs.show-diff}} --go-version=${GO_VERSION} --compat=${{inputs.tidy-compat}}
+      gobump ${packagesString} ${replacesString} --tidy=${{inputs.tidy}} --skip-initial-tidy=${{inputs.skip-initial-tidy}} --show-diff=${{inputs.show-diff}} --go-version=${GO_VERSION} --compat=${{inputs.tidy-compat}}
```

Comment:
This will have quoting issues if there are more than one "deps" won't it?

Reply:
go bump ${packageString:+"--packages=${{packageString}"}